api/base/filters.py

```diff
@@ -24,17 +24,30 @@
 def sort_multiple(fields):
     fields = list(fields)
     def sort_fn(a, b):
-        while fields:
-            field = fields.pop(0)
-            a_field = getattr(a, field)
-            b_field = getattr(b, field)
-            if a_field > b_field:
-                return 1
-            elif a_field < b_field:
-                return -1
+        for field in fields:
+            if field[0] == '-':
+                field = field[1:]
+                a_field = getattr(a, field)
+                b_field = getattr(b, field)
+                if a_field > b_field:
+                    return -1
+                elif a_field < b_field:
+                    return 1
+                elif a_field == b_field:
+                    return 1
```

Comment:
Do you really mean `1` here? Usually when comparing, equality results in `0`, as in line 47.